File: Python/OrdenandoEnCubetas.py
def main():

    cantidad_de_pelotas = int(input())
    maximo_numero_a_encontrar = int(input())

    pelotas = [0] * (maximo_numero_a_encontrar + 1)

    if maximo_numero_a_encontrar <= 100:
        for _ in range(cantidad_de_pelotas):
            num_pelotas = int(input())
            pelotas[num_pelotas] += 1

            if pelotas[num_pelotas] > 100:
                print("El numero de pelota exede el rango.")
                break

        for i in range(1, maximo_numero_a_encontrar + 1):
            print(f"{i}: {pelotas[i]}")

main()


# https://omegaup.com/arena/problem/Ordenando-en-cubetas/

File: C++/OrdenandoEnCubetas.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int cantidadDePelotas, maximoNumeroAEncontrar, numPelotas;

    cin >> cantidadDePelotas;
    cin >> maximoNumeroAEncontrar;
    
    vector<int> pelotas(cantidadDePelotas + 1);

    if (maximoNumeroAEncontrar < 101)
    {
        for (int i = 0; i < cantidadDePelotas; i++)
        {
            cin >> numPelotas;
            pelotas[numPelotas]++;

            if (pelotas[numPelotas] > 100)
            {
                cout << "El numero de pelotas exede el rango. ";
                break;
            }
        }

        for (int i = 1; i <= maximoNumeroAEncontrar; i++)
        {
            cout << i << ": " << pelotas[i] << endl;
        }
    }

    return 0;
}

// https://omegaup.com/arena/problem/Ordenando-en-cubetas/